src/printer.js

```diff
@@ -334,10 +334,10 @@ function genericPrintNoParens(path, options, print) {
       ])));
     }
 
-    parts.push(" => ");
+    parts.push(" =>");
 
     return conditionalGroup([
-      concat([ concat(parts), path.call(print, "body") ]),
+      concat([ concat(parts.concat([" "])), path.call(print, "body") ]),
```

Comment:
You can just do `concat([ concat(parts), " ", path.call(print, "body") ])` which removes that `.concat` call